docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,33 +19,42 @@
   },
```

Comment:
<p>If switching to from_pandas, should match that in this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done.</p>